app/services/filter_service.rb

```diff
@@ -17,6 +17,14 @@ def self.filter_logs(logs, search_term, filters, all_orgs, user)
       logs = logs.public_send("filter_by_#{category}", values, user)
     end
     logs = logs.order(created_at: :desc)
-    user.support? ? logs.all.includes(:owning_organisation, :managing_organisation) : logs
+    if user.support?
+      if logs.first.instance_of?(LettingsLog)
```

Comment:
(Assuming that James's [suggestion](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1169/files#r1067915639) works.) 
```suggestion
      if logs.first.lettings?
```